mapper/mapper_test.go

```diff
@@ -580,6 +581,50 @@ func TestUnitMapper(t *testing.T) {
 			})
 		})
 	})
+
+	Convey("FuncLookupSection", t, func() {
```

Comment:
I would suggest using the `Given, When, Then, And` convention when writing the tests. This will clearly inform the situation and the end result for each scenario

Reply:
Is there something unclear about the tests in their current form, or is the request borne of a desire to mimic the approach in the existing tests?